<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BrowserWindow</class>
 <widget class="QMainWindow" name="BrowserWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>637</width>
    <height>561</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>App Swipe</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color: rgb(255, 255, 240);
font: 13pt &quot;Roboto&quot;;
background-color: rgb(0, 0, 0);

selection-color: rgb(255, 255, 240);
selection-background-color: rgb(0, 100, 0)
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"> QPushButton:pressed {
     border: none 0px;
     border-radius: 0px;
	 background-color: #80000080;
 }
/*QPushButton {
     border: none;
     border-radius: 0px;
	 background-color: #00008080;
 }
QPushButton:hover {
     border: none;
	 background-color: #00800040;
}

*/
/*
 QPushButton:flat {
     border: none; 
 }
*/

QMenu 
{
	background-color: #393939;
}

QTabBar::tab
{
    background: #393939; 
}

QTabBar::tab:selected 
{
	background-color: #151510; 
}

        /*********** horizontal scroll bar **********/
        QScrollBar:horizontal
        {
            background: #00000000;

            height: 16px;

            border: 1px solid #808080;
            border-top-left-radius: 7px;
            border-top-right-radius: 7px;
            border-bottom-left-radius: 7px;
            border-bottom-right-radius: 7px;
        }

        QScrollBar::handle:horizontal
        {
            background: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:1 #2040E1, stop:0 #4080FF);
            min-width: 45px;
            border: 0px solid #32CCFF;
            border-top-left-radius: 5px;
            border-top-right-radius: 5px;
            border-bottom-left-radius: 5px;
            border-bottom-right-radius: 5px;
            margin: 1px
        }

        /* this is the one line up button */
        QScrollBar::sub-line:horizontal
        {
            background: none;
            width: 0px;
            height: 0px;
        }

        /* this is the one line down button */
        QScrollBar::add-line:horizontal
        {
            background: none;
            width: 0px;
            height: 0px;
        }
/* vertical scroll bar **************************/

QScrollBar:vertical
        {
            background: #000000;
/*            background: #A0EFEFFF;*/

            width: 16px;

            border: 1px solid #808080;
            border-top-left-radius: 7px;
            border-top-right-radius: 7px;
            border-bottom-left-radius: 7px;
            border-bottom-right-radius: 7px;
        }

        QScrollBar::handle:vertical
        {
            background: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:1 #2040E1, stop:0 #4080FF);
            min-height: 45px;
            border: 0px solid #32CCFF;
            border-top-left-radius: 5px;
            border-top-right-radius: 5px;
            border-bottom-left-radius: 5px;
            border-bottom-right-radius: 5px;
            margin: 1px
        }

        /* this is the one line up button */
        QScrollBar::sub-line:vertical
        {
            background: none;
            width: 0px;
            height: 0px;
        }

        /* this is the one line down button */
        QScrollBar::add-line:vertical
        {
            background: none;
            width: 0px;
            height: 0px;
        }


</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="toolbar" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>45</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 240);
font: 16pt &quot;Roboto&quot;;
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                  stop: 0.55 #000000, stop: 0.90 #004080, stop: 0.98 #1080A0,
                                  stop: 0.99 #22A4D4, stop: 1.0 #33B5E5);
/*
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                  stop: 0.5 #000000, stop: 0.7 #004000, stop: 0.9 #008000,
                                  stop: 0.99 #108010, stop: 1.0 #708070);
*/
/* rgb(0, 0, 0); */

selection-color: rgb(255, 255, 240);
/* selection-background-color:  qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                  stop: 0.2 #000000, stop: 0.8 #008000,
                                  stop: 0.9 #108010, stop: 1.0 #708070); */
selection-background-color:  #007000;
/* rgb(0, 80, 0); */
</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="newTabButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Open new tab</string>
         </property>
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/img/ic_highlight_plus_48px.svg</normaloff>:/img/ic_highlight_plus_48px.svg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="backButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Back</string>
         </property>
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/img/ic_arrow_back_48px.svg</normaloff>:/img/ic_arrow_back_48px.svg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="forwardButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Forward</string>
         </property>
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/img/ic_arrow_forward_48px.svg</normaloff>:/img/ic_arrow_forward_48px.svg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="reloadButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Refresh</string>
         </property>
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/img/ic_refresh_48px.svg</normaloff>:/img/ic_refresh_48px.svg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="frame">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QProgressBar" name="searchProgress">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="clipButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Save to clipboard instead of executing</string>
         </property>
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/img/clipboard.svg</normaloff>:/img/clipboard.svg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="askButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Ask before attempting system changes</string>
         </property>
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/img/jean_victor_balin_unknown_green.svg</normaloff>:/img/jean_victor_balin_unknown_green.svg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="menuButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/img/appicon.svg</normaloff>:/img/appicon.svg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>36</width>
           <height>36</height>
          </size>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="TabWidget" name="tabWidget">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::West</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>-1</number>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
      <property name="elideMode">
       <enum>Qt::ElideRight</enum>
      </property>
      <property name="documentMode">
       <bool>true</bool>
      </property>
      <property name="tabBarAutoHide">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TabWidget</class>
   <extends>QTabWidget</extends>
   <header>tabwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
